### DISCUSSION TITLE
Merging tables with 34million rows in Python

### DISCUSSION THREAD

**Discussion by JCris04** [codewars/codewars.com#2878] [Q&A]
Hi all,

I am still learning coding but I have to merge 2 tables, 1 with 34M rows and I cannot find a code to do so. Lost where to find it.
I tried simple ways I find in the web but not working for me.
![merging problem](https://user-images.githubusercontent.com/117014891/206797908-b4b6e219-46e8-4819-8552-352b5fa3ea1e.JPG)
thanks

also tried using this code but syntax error:

def merge(
    pick,
    product,
    how="left",
    on=SKU,
    left_on=None,
    right_on=None,
    left_index=False,
    right_index=False,
    sort=False,
    suffixes=("_x", "_y"),
    copy=True,
    indicator=False,
    validate=None,
)

**Comment by hobovsky** [comment]
Github Discussions are not a proper place to discuss homework questions. This place is meant to discuss and exchange ideas related to Codewars platform.
